NitroxServer/Serialization/BatchCellsParser.cs

```diff
@@ -44,7 +44,14 @@ public List<EntitySpawnPoint> ParseBatchData(Int3 batchId)
             ParseFile(batchId, "", "creatures", spawnPoints);
             ParseFile(batchId, "", "other", spawnPoints);
 
-            Log.Debug($"Loaded {spawnPoints.Count} entity-spawn-points for batch {batchId}");
+            if (spawnPoints.Count > 0)
```

Comment:
I think the old way was fine.  This was logged as a debug message.  I think we should toggle off debug messages for a release build.